### PR TITLE
Bump NFD to V0.7.0

### PR DESCRIPTION
**Pull request by ArangoGutierrez**
Signed-off-by: Carlos Eduardo Arango Gutierrez <carangog@redhat.com>

### CODE REVIEW

**Comment by ArangoGutierrez** [comment]
Depends on: https://github.com/kubernetes-sigs/node-feature-discovery/issues/394

**Comment by ArangoGutierrez** [comment]
https://github.com/kubernetes-sigs/node-feature-discovery/pull/414
/unhold

File: manifests/0700_cr.yaml [hunk 1]
```diff
@@ -6,7 +6,7 @@ metadata:
 spec:
   operand:
     namespace: node-feature-discovery-operator
-    image: k8s.gcr.io/nfdnode-feature-discovery:v0.6.0
+    image: k8s.gcr.io/nfdnode-feature-discovery:v0.7.0
```
Comment (marquiz) [inline manifests/0700_cr.yaml:9]: Missing slash here
```suggestion
    image: k8s.gcr.io/nfd/node-feature-discovery:v0.7.0
```

Comment (marquiz) [reply manifests/0700_cr.yaml:9]: Hmm, even the previous version was borken 🤔

**Pull request merged**